web/src/stream_settings_ui.js

```diff
@@ -482,6 +482,7 @@ export function maybe_reset_right_panel() {
         $(".right .settings").hide();
         $(".nothing-selected").show();
         $(".stream-row.active").removeClass("active");
+        $("#subscription_overlay .stream-info-title").text($t({defaultMessage: "Stream settings"}));
```

Comment:
The title is updated correctly but the hash still remains incorrect. Also, this code is mostly same as `open_edit_panel_empty` so, we can probably deduplicate the code as well by re-using that here. Note that #28460 has some related changes and would conflict with this PR.